web3.js/src/connection.ts

```diff
@@ -867,6 +867,10 @@ export type ConfirmedTransactionMeta = {
   preTokenBalances?: Array<TokenBalance> | null;
   /** The token balances of the transaction accounts after processing */
   postTokenBalances?: Array<TokenBalance> | null;
+  /** The addresses of the accounts loaded for the transaction */
+  loadedAddresses?: LoadedAddresses | null;
```

Comment:
Missed a spot from #27065, @jstarry?

Reply:
Yup it was missed but recently added in https://github.com/solana-labs/solana/pull/27068, this addition just broke master unfortunately. In the future let's keep unrelated changes out of PR's

Reply:
Haha. Scroll down, @steveluscher. Sorry about that.

Reply:
Oh no worries! Merge conflicts happen, and it was originally my fault for missing that field :)